mne/commands/utils.py

```diff
@@ -4,14 +4,29 @@
 #
 # License: BSD (3-clause)
 
-import imp
+import sys
 import os
 import re
 from optparse import OptionParser
 
 import mne
 
 
+def load_module(name, path):
+    if sys.version_info < (3, 3):
+        import imp
+        return imp.load_source(name, path)
```

Comment:
I'll try `importlib.machinery.SourcelessFileLoader`, at least this sounds like it could work...